docusaurus.config.js

```diff
@@ -161,11 +161,11 @@ const darkCodeTheme = require('prism-react-renderer/themes/dracula');
           label: 'Doc',
           position: 'right',
           items: [
-            {label: '1.4.0', to: '/docs/latest/about/introduction'},
-            {label: '1.3.2', to: '/docs/1.3.2/about/introduction'},
-            {label: '1.3.1', to: '/docs/1.3.1/about/introduction'},
-            {label: '1.3.0', to: '/docs/1.3.0/introduction'},
-            {label: '1.2.0', to: '/docs/1.2.0/introduction'},
+            // {label: '1.4.0', to: '/docs/latest/about/introduction'},
+            // {label: '1.3.2', to: '/docs/1.3.2/about/introduction'},
+            // {label: '1.3.1', to: '/docs/1.3.1/about/introduction'},
+            // {label: '1.3.0', to: '/docs/1.3.0/introduction'},
+            // {label: '1.2.0', to: '/docs/1.2.0/introduction'},
             {label: '1.1.1', to: '/docs/1.1.1/introduction'},
```

Comment:
The latest two versions（1.4.0 and 1.3.2） are retained
It is also necessary to update the overview information here(https://deploy-preview-761--apache-linkis-website.netlify.app/versions) at the same time (all document versions need to be listed here) 
![image](https://github.com/apache/linkis-website/assets/7869972/323a75e3-f296-44b5-9f1b-096808b49e42)